Hackney.Core/Constants.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Hackney.Core
+{
+    public static class Constants
```

Comment:
Would we not want to store the file in the middleware folder?

Reply:
Fair point. I will move it and name it better.